CMakeLists.txt

```diff
@@ -44,22 +44,38 @@ if(APPLE)
   set(CMAKE_CXX_FLAGS "-stdlib=libc++")
 endif()
 
+set(CMAKE_C_FLAGS ${CMAKE_C_FLAGS} "-m64 -pipe -O2 -Wall -W -fPIE")
+set(CMAKE_CXX_FLAGS ${CMAKE_CXX_FLAGS} "-m64 -pipe -fno-strict-aliasing -frounding-math -std=c++0x -O2 -std=c++0x -Wall -W -Wno-unused-local-typedefs -fPIE")
+add_definitions(-DSTACKSIZE=8388608 -D_REENTRANT)
+
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules/")
 set(CMAKE_PREFIX_PATH "$ENV{OPENSCAD_LIBRARIES}")
 
 find_package(Eigen3 REQUIRED QUIET)
 message(STATUS "Eigen3: ${EIGEN3_VERSION_STRING}")
 include_directories(${EIGEN3_INCLUDE_DIRS})
-
-find_package(Boost 1.36 REQUIRED COMPONENTS filesystem system thread regex program_options QUIET)
-message(STATUS "Boost: ${Boost_MAJOR_VERSION}.${Boost_MINOR_VERSION}.${Boost_SUBMINOR_VERSION}")
-include_directories(${Boost_INCLUDE_DIRS})
+add_definitions(-DEIGEN_DONT_ALIGN)
```

Comment:
What does this do? It's a deprecated flag to Eigen

Reply:
I don't know what it does, but it is in the `Makefile` and was before not in the CMake build file. You think we should rather remove it from the Makefile?